spec/requests/api/blueprints_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe "Blueprints API" do
+  class Blueprint
```

Comment:
@bzwei I created this stub for my tests - can you confirm that it will have the same basic interface?